### PR TITLE
Random egoist stats

### PR DESCRIPTION
**Pull request by LanternRoman**
## About The Pull Request

Adds random stats for the current egoists. Higher grade distortions have a better chance to spawn with better stats than weaker distortions.

## Why It's Good For The Game

It adds randomization of egoist stats, meaning they don't always spawn with the same stats.

## Changelog
:cl:
add: Added randomization of egoist stats.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/distortion/myth/another_day_work.dm [hunk 1]
```diff
@@ -38,9 +38,17 @@
...
 	egoist_outfit = /datum/outfit/job/civilian
+	//The Egoist's starting stats, all across the board. MUST BE AT LEAST THE MINIMUM TO EQUIP THE EGO_GEAR IF ANY
+	egoist_attributes = 40
 	//Loot on death; distortions should be valuable targets in general.
 	loot = list(/obj/item/documents/ncorporation)
 
+//Set random stats for the egoist
+/mob/living/simple_animal/hostile/distortion/another_day_work/Initialize()
+    . = ..()
+    egoist_attributes = min(rand(0, 130), rand(0, 130))
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/distortion/myth/another_day_work.dm:49]: Change the 0 to egoist_attributes

File: code/modules/mob/living/simple_animal/distortion/plague/bunnyman.dm [hunk 2]
```diff
@@ -45,6 +45,12 @@
 	var/can_act = TRUE
 	var/transformed = FALSE
 
+//Set random stats for the egoist
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/distortion/plague/bunnyman.dm:48]: After that's done, you can remove this, and just change the egoist_attributes variables

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Minor code cleanups that help.

File: code/modules/mob/living/simple_animal/distortion/myth/another_day_work.dm (hunk 1 above)
Comment (LanternRoman) [reply code/modules/mob/living/simple_animal/distortion/myth/another_day_work.dm]: Alright, I changed it. Should be alright like that I guess.

**Comment by Coxswain-Navigator** [comment]
I don't think this PR is even necessary anymore.

**Pull request closed without merging**